libsql-server/tests/cluster/replication.rs

```diff
@@ -91,7 +91,7 @@ fn apply_partial_snapshot() {
     sim.client("client", async move {
         let primary = libsql::Database::open_remote_with_connector(
             "http://primary:8080",
-            "",
+            "dummy_token",
```

Comment:
is there a problem with using the old `""`? I know we have this like everywhere with examples and our own code so ensuring that we can support that old method I think is quite important.

Reply:
parser was crashing on string split
I was meaning to ask - what is the actual behavior we expect?
if auth header is present but not required, do we just ignore its content?